api/src/controllers/put/mine-permit.js

```diff
@@ -0,0 +1,74 @@
+const mongoose = require('mongoose');
```

Comment:
Do we still have the concept of a `mine-permit`, or is this replaced by `permit` and `permt-amendment`?

Reply:
Do not. Removed.